default.nix

```diff
@@ -8,6 +8,9 @@ if ! builtins ? nixVersion || builtins.compareVersions requiredVersion builtins.
 
     - If you are running NixOS, use `nixos-rebuild' to upgrade your system.
 
+    - Alternatively, with Nix > 2.0 you can use `nix upgrade-nix' to imperatively
```

Comment:
"you can use" to "can be used" in both lines

Reply:
Done.